src/dom.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 /* eslint max-lines: off */
-
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Is `uuid` a new dependency? I'm concerned with adding a new dependency to belter.